test/gtest/ucs/test_rcache.cc

```diff
@@ -841,7 +841,8 @@ class test_rcache_pfn : public ucs::test {
 
         status = ucs_sys_enum_pfn((uintptr_t)address, page_num, enum_pfn_cb, &ctx);
         ASSERT_UCS_OK(status);
-        ASSERT_TRUE(ctx.page.size() == ctx.pfn.size());
+        ASSERT_TRUE(ctx.page.size() == page_num);
+        ASSERT_TRUE(ctx.pfn.size() == page_num);
```

Comment:
EXPECT_EQ

Reply:
also add comment that we expect that every page will have a unique page_num and a unique PFN

Reply:
fixed